core/project.clj

```diff
@@ -1,4 +1,4 @@
-(defproject jarl "0.1.0-SNAPSHOT"
+(defproject by.borge.jarl/jarl "0.1.0-SNAPSHOT"
```

Comment:
Not sure if this breaks anything for Clojure consumers, but it seems to be needed to get a proper groupId in `pom.xml`.